<launch>
    <group ns="aleph1">
        <param name="robot_description" command="xacro $(find aleph_viz)/urdf/aleph.urdf.xacro" />
        <param name="robot_description_semantic" textfile="$(find manip_moveit_config)/config/aleph.srdf" />
    </group>

    <group ns="aleph2">
        <param name="robot_description" command="xacro $(find aleph2_description)/urdf/aleph2.urdf.xacro"/>
    </group>
</launch>